test/config_test.py

```diff
@@ -369,4 +391,7 @@ def test_user_token_supersedes_bearer_token(self):
         get_request = requests.Request("GET", "http://foo:5555/get")
         prepared_get = connection.session.prepare_request(get_request)
         self.assertFalse("authorization" in prepared_get.headers)
-        self.assertTrue("X-User-Email" in prepared_get.headers)
+        self.assertTrue(
+            prepared_get.headers["x-organization-id"] == connection.organization_id
+        )
+        self.assertTrue(prepared_get.headers["x-user-email"] == connection.email)
```

Comment:
See above, would prefer all the casing here to be consistent

Reply:
Updated to be consistent